decentraland/decentraland-gatsby
# fix: wallet connect sign

# How to test?

- At sign in, wallet connect signature must show text showing the ephemeral address instead of hex string
- At sign in, metamask (injected) should work showing the ephemeral address as well
